src/Mvc/.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Let's remove this `settings.json` file for now and retain a `launch.json` and `tasks.json` file in each of the subdirectories.